apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -914,6 +914,10 @@ async function renderPanel(
       locals.urlPrefix,
     ),
     submission_files_url: submission ? submissionFilesUrl : null,
+
+    variant_id: variant?.id,
```

Comment:
Nit, `variant` should always be defined so optional chaining isn't necessary and could in fact be confusing:

```suggestion
    variant_id: variant.id,
```